src/typeParser.test.ts

```diff
@@ -0,0 +1,52 @@
+import * as ts from 'typescript'
+
+import {TypeParser} from './typeParser'
+import {TypePrettyPrinter} from './typePrettyPrinter'
+import {ArrayType, NamedType, numberType, ObjectType, Type} from './types'
+
+const INLINE_MODULE = 'inline.ts'
+
+function parseTypeFromSource(typeName: string, source: string): NamedType {
```

Comment:
this probably makes sense for other integration tests that start from parsing but also test other pieces. perhaps move it out of here? on demand?

Reply:
on demand